development/workflow.rst

```diff
@@ -0,0 +1,667 @@
+   If you are uncertain about who the assignee should be you can allow the ticket to be automatically assigned (which defaults to the component's T/CAM; `RFC-51 <https://jira.lsstcorp.org/browse/RFC-51>`_).
+Description
+   For stories, you should specify the ticket's scope and outline your implementation design in this field.
+   For bug reports, include any information needed to diagnose and reproduce the issue.
+   Feel free to use `Atlassian markup syntax <https://jira.lsstcorp.org/secure/WikiRendererHelpAction.jspa?section=texteffects>`_. 
+Story Points
+   Use this field, at ticket creation time, to **estimate** the amount of effort involved to accomplish the work.
+   Keep in mind how *velocity* (see above) converts story points into real-world days.
+Labels
+   Think of labels as tags that you can use to sort your personal work.
+   Unlike the Component and Epic fields, you are free to create and use labels in any way you see fit.
+Linked Issues
+   You can express relationships between JIRA issues with this field.
+   For example, work that implements an RFC should link to that RFC.
+   You can also express dependencies to other work using a 'is Blocked by' relationship.
+Epic Link
+   If the ticket is a story, you must specify what epic it belongs to with this field.
+   By definition, bug or enhancement-type tickets are not associated with an epic.
+Team
+   You must specify which DM team doing the work with this field, for accounting purposes.
+   The owner of the epic should be consistent with the Team working on a ticket.
+
+.. _workflow-jira-ticket-status:
+
+Ticket status
+-------------
+
+Tickets are created with a status of 'Todo.'
+
+If a ticket was assigned to you by someone else, you can set the status to 'Acknowledged' once you agree to schedule the work.
+
+Once a ticket is being actively worked on you can upgrade the ticket's status to 'In Progress.'
+
+.. _git-branching:
+
+DM Git Branching Policy
+=======================
+
+Rather than forking LSST's GitHub repositories, DM developers use a *shared repository model* by cloning repositories in the `lsst <https://github.com/lsst>`_, `lsst-dm <https://github.com/lsst>`_, and `lsst-sqre <https://github.com/lsst>`_ GitHub organizations.
+Since the GitHub ``origin`` remotes are shared, it is essential that DM developers adhere to the following naming conventions for branches.
+
+See `RFC-21 <https://jira.lsstcorp.org/browse/RFC-21>`_ for discussion.
+
+.. _git-branch-integration:
+
+The master branch
+-----------------
+
+``master`` is the main integration branch for our repositories.
+The master branch should always be stable and deployable.
+In some circumstances, a ``release`` integration branch may be used by the release manager.
+Development is not done directly on the ``master`` branch, but instead on *ticket branches*.
+
+Documentation edits and additions are the only scenarios where working directly on ``master`` and by-passing the code review process is permitted.
```

Comment:
Well, trivial (like one-character) fixes are allowed to be done that way as well, but maybe it's better not to mention it.